### PR TITLE
New structure

### PR DESCRIPTION
**Pull request by Hladikes**
I want to implement grid/list toggle button, but before that, I wanted to give this project a bit of a structure.
What do you think meta-master ?

### CODE REVIEW

File: web/js/components/CardsContainer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Card from './Card.js'
+
+export default async function CardsContainer() {
+  const response = await fetch(
+  'https://api.github.com/repos/vikiival/trap/contents/images?ref=master'
```
Comment (vikiival) [inline web/js/components/CardsContainer.js]: Why does it fetch it from the repo when the images are stored locally?

**Review by vikiival — CHANGES_REQUESTED** [review]
Smol stuff
Otherwise oki

File: web/js/components/CardsContainer.js (hunk 1 above)
Comment (Hladikes) [reply web/js/components/CardsContainer.js]: @vikiival these files are just statically hosted, and right now, I am not aware of any way we can list all available images in the folder 🤔

**Review by vikiival — APPROVED** [review]
My bad, 

LGTM

**Comment by Hladikes** [comment]
image

**Pull request merged**